audits/202402_Certora.md

```diff
@@ -0,0 +1,40 @@
+# 2024-02 Certora Security Audit & Formal Verification
+
+## Audit Dates
+
+December 3rd, 2023 -> February 8th, 2024.
+
+## Audit Scope
+
+Full protocol audit including all smart contracts as per commit [0b15fbfa36af17c1f5e8059de40c2de325210204](https://github.com/liquid-collective/liquid-collective-protocol/commit/0b15fbfa36af17c1f5e8059de40c2de325210204)
+
+It covers the following smart contracts & all the contracts that they inherit:
+- River
+- Oracle
+- Allowlist
+- Withdraw
+- ELFeeRecipient
+- WLSETH
+- Firewall
+- TUPProxy
+- Initializable
+- TLC
+- OperatorsRegistry
+- CoverageFund
+- Administrable
+- RedeemManager
+
+## Audit Summary
+
+|    **Severity**   | **Count** | **Fixed** | **Acknowledged** |
+|:-----------------:|:---------:|:---------:|:----------------:|
+|   Critical Risk   |     1     |     1     |         1        |
+|     High Risk     |     2     |     2     |         1        |
+|    Medium Risk    |     4     |     4     |         2        |
+|      Low risk     |     5     |     4     |         2        |
+|   Informational   |     3     |     3     |         1        |
+|     **Total**     |     15    |     14    |         7        |
+
+## Audit Report
+
+See [full report]() for more details.
```

Comment:
@iamsahu do we not have the URL for the full report online yet ?

I thought the folk at certora have published the report already ?

Reply:
They had accidentally published the DI report. They are yet to publish the LC report.